java-checks/src/main/java/org/sonar/java/checks/DeprecatedHashAlgorithmCheck.java

```diff
@@ -39,168 +35,26 @@
 import org.sonar.plugins.java.api.tree.NewClassTree;
 import org.sonar.plugins.java.api.tree.Tree;
 
-import static org.sonar.java.checks.DeprecatedHashAlgorithmCheck.InsecureAlgorithm.MD2;
-import static org.sonar.java.checks.DeprecatedHashAlgorithmCheck.InsecureAlgorithm.MD5;
-import static org.sonar.java.checks.DeprecatedHashAlgorithmCheck.InsecureAlgorithm.SHA1;
-import static org.sonar.plugins.java.api.semantic.MethodMatchers.ANY;
```

Comment:
Importing statically in one file and using a qualified name in another feels like cheating the duplication. In my opinion, we should either accept it or organize the code to really avoid duplication. 
In this case, isn't it possible to create an abstract class instead of a helper, and having two rules inheriting from it?

Reply:
I thought about it, but in the future we'll remove one of the rule, so we'll have only one class. Actually I can refactor it, so we won't have duplications without this trick